drogon_config.json

```diff
@@ -1,16 +1,50 @@
 {
-	"log": {
-		"log_level" : "INFO"
-	},
-	"listeners": [
-		{
-			"address": "0.0.0.0",
-			"port": 4231,
-			"https": false
-		}
-	],
-	"app": {
-		"document_root": "../content",
-		"enable_session": false
-	}
-}
\ No newline at end of file
+    "listeners": [
+        {
+            "address": "0.0.0.0",
+            "port": 80,
```

Comment:
Let's make it listen on a non-privileged port by default. Otherwise it's annoying for development. And confuses new users about why their build doesn't work.